src/main/scala/is/hail/utils/richUtils/RichRDDByteArray.scala

```diff
@@ -8,30 +8,48 @@ import org.apache.spark.rdd.RDD
 import scala.reflect.ClassTag
 
 class RichRDDByteArray(val r: RDD[Array[Byte]]) extends AnyVal {
-  def saveFromByteArrays(filename: String, tmpDir: String, header: Option[Array[Byte]] = None, deleteTmpFiles: Boolean = true) {
+  def saveFromByteArrays(filename: String, tmpDir: String, header: Option[Array[Byte]] = None,
+    deleteTmpFiles: Boolean = true, parallelWrite: Boolean = false) {
     val nullWritableClassTag = implicitly[ClassTag[NullWritable]]
     val bytesClassTag = implicitly[ClassTag[BytesOnlyWritable]]
     val hConf = r.sparkContext.hadoopConfiguration
 
-    val tmpFileName = hConf.getTemporaryFile(tmpDir)
+    hConf.delete(filename, recursive = true) // overwriting by default
```

Comment:
In VDS write we normally throw an error when writing to an existing file. Why is overwriting the right default here?

Reply:
I made it symmetric to `RichRDD.writeTable` which also overwrites by default. I can make a separate PR to change the behavior for both methods if needed.